client/src/pages/Home.tsx

```diff
@@ -168,9 +184,10 @@ function Home(this: any) {
       <hr className="h-px mt-2 border-0 bg-gray-300"></hr>
       <div className="flex justify-between mt-10 mx-60">
         <p className="flex items-baseline justify-start space-x-4 text-xl">
-          Sentiment analysis of the text is:
+          {sentimentTextPlaceholder}
           <span style={setSentimentStyle()} id="sentiment-result">
-            {sentimentText} {`${Number(sentimentScore)}%`}
+            {sentimentText !== "" &&
+              `${sentimentText} ${sentimentScore}${percent}`}
```

Comment:
The `percent` isn't coming through on the UI. The Cypress test is also failing because it assumes that the value has a percentage symbol (%) sign next to it.